### PR TITLE
Added the feature of sharing the images with Tumblr 

### PR DESCRIPTION
**Pull request by VarthanV**
**Description**
        I have been using the extension for a while and I have been sharing images regularly from the extension. I use Tumblr regularly to share my thoughts whenever  I have to share the image it becomes difficult for me since the extension doesn't have an option to share it in Tubmlr directly as it has for the other platforms. So i implemented the feature of sharing the image directly to the Tumblr platform. 

Screenshot 2019-11-27 at 6 49 10 PM

Screenshot 2019-11-27 at 7 15 03 PM

### CODE REVIEW

File: src/popup/infoPopupModule.js [hunk 1]
```diff
@@ -127,26 +130,31 @@ function getPinterestShareLink(sourceLink, imageLink) {
   return `https://pinterest.com/pin/create/button/?url=${sourceLink}&media=${imageLink}&description=I%20found%20an%20image%20through%20CC%20search%20%40creativecommons%3A%20${sourceLink}`;
 }
 
+// Added the share link for tumblr
+function getTumblrShareLink(sourceLink,imageLink) {
```
Comment (makkoncept) [inline src/popup/infoPopupModule.js]: Require space after the `,`. [comma-spacing](https://eslint.org/docs/rules/comma-spacing)

File: src/popup/infoPopupModule.js [hunk 2]
```diff
@@ -1,72 +1,73 @@
...
   return `<p style="font-size: 0.9rem;font-style: italic;">${imgLink}${creator}${licenseLink}${licenseImgLink}</p>`;
 }
 
 export function getPlainAttribution(image) {
   if (!image) {
-    return '';
+    return "";
   }
-  let creatorUrl = 'Not Available';
+  let creatorUrl = "Not Available";
```
Comment (makkoncept) [inline src/popup/infoPopupModule.js]: The string must use single quotes.

**Review by makkoncept — CHANGES_REQUESTED** [review]
Hi @VarthanV. Thanks for this PR.
I have checked the changes locally and everything is working fine. But the changes are violating eslint rules and therefore, are giving linting errors. I have marked a couple of them here. Please fix those. 

You can run `npm run lint` to run the lint. Or you can also install an Eslint extension for your editor to view the errors and warnings.

**Comment by VarthanV** [comment]
Sir Thank you four your review, I  have made the necessary changes sir by using the methods that you suggested above . Thank You!

**Comment by VarthanV** [comment]
Sorry for the Inconvenience sir, I have reviewed it and fixed all the eslint errors! Thank You!

**Comment by VarthanV** [comment]
Sir I have made the changes will my commit be merged?

**Comment by makkoncept** [comment]
@VarthanV you need to remove the "Default Filters" heading on the options page. There is no need of that.

**Comment by VarthanV** [comment]
Sir I have removed the default filters heading  please do review my change

**Comment by makkoncept** [comment]
@VarthanV you have to also push the changes, because they are not showing in this PR.
The last commit from your side has been "Fixed ESLint errors".

**Comment by VarthanV** [comment]
I  have made the necessary changes @makkoncept

**Comment by makkoncept** [comment]
@VarthanV you still don't have removed the "Default filters" heading in `options.html`. Also, please write better commit message.

**Comment by VarthanV** [comment]
@makkoncept  sir .I have made the changes for sure now and added a valid Commit message

**Pull request merged**